app/components/personal_statement_component.html.erb

```diff
@@ -6,30 +6,17 @@
     </span>
   </summary>
   <div class="govuk-details__text">
-    <h2 class="govuk-heading-m">Why do you want to be a teacher?</h2>
-    <p class="govuk-body">Use this section to showcase your motivation, commitment and teaching potential, backing up your answer with specific examples.</p>
-    <p class="govuk-body">Give providers an insight into your personality by writing honestly and thoughtfully. Avoid cliché and write in clear, correct, concise English.</p>
-    <p class="govuk-body">You can ask your training provider for examples written by successful applicants, or register with <a href="https://beta-getintoteaching.education.gov.uk/" class="govuk-body">Get into Teaching</a> for help from a teacher training adviser.</p>
-    <p class="govuk-body govuk-!-font-weight-bold">Suggested topics to cover include:</p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>why you want to be a teacher</li>
-      <li>your passion for your subject and the age group you’ve chosen to teach</li>
-      <li>the welfare and education of children and/or young people</li>
-      <li>the demands and rewards of the profession</li>
-      <li>personal qualities that will make you a good teacher</li>
-      <li>your contribution to the life of the school outside the classroom – for example, running extra-curricular activities and clubs</li>
-      <li>if you have school experience or have worked as a volunteer with children or young people, give details of what this has taught you</li>
-    </ul>
-    <h2 class="govuk-heading-m">What do you know about the subject you want to teach?</h2>
-    <p class="govuk-body">Give us detailed evidence for the knowledge and interest you bring to your course choice.</p>
-    <p class="govuk-body">Evidence can include:</p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>the subject of your undergraduate degree</li>
-      <li>modules you studied as part of your degree</li>
-      <li>postgraduate degrees (for example, a Masters or PhD)</li>
-      <li>your A level subjects</li>
-      <li>expertise you’ve gained at work</li>
+    <h2 class="govuk-heading-m"><%= t('page_titles.becoming_a_teacher') %></h2>
+    <p class="govuk-body">You can include:</p>
+      <ul class="govuk-list govuk-list--bullet">
+      <li>your interest in the subject or age group</li>
+      <li>the demands and rewards of teaching</li>
+      <li>the personal qualities that would make you a good teacher</li>
+      <li>how you could contribute to a school outside of the classroom</li>
+      <li>any past experience working with children or young people, and what you learnt</li>
+      <li>your thoughts on welfare and education</li>
     </ul>
+    <p class="govuk-body">If you need help, you can speak to a <%= govuk_link_to t('service_name.get_into_teaching'), t('get_into_teaching.url_get_an_advisor') %> adviser. They’re all experienced teachers who can give you advice on applying for teacher training.</p>
```

Comment:
The content shown in the provider interface has fallen out of step with what’s shown to candidates. Not the first time this has happened, either. Surely there must be a better way to keep these in sync… maybe for starters we can add some comments in the code? @duncanjbrown

Reply:
This is a fact of life rn unfortunately, unless we keep track of all previous versions — applications submitted in the past ought to have the guidance from then, rather than now. We accepted this as a drawback when we did this in the first place, but we should def update this in the provider interface too now.

Reply:
Ah, the whole historical aspect. Gah!

> we should def update this in the provider interface too now

Just to be clear, that’s what this PR does.

Reply:
Having spoken to the ProVendor design team, we’re actually remove this guidance. So in this PR I’ll just limit the change to pointing to the new GIT constants. Much better.